test/src/templates/big_fact_slide_test.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_deck/flutter_deck.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'test_utils.dart';
+
+void main() {
+  group('BigFactSlide', () {
+    testWidgets('should render all layout elements',
+        (WidgetTester tester) async {
+      final slideTester = SlideTester(
+        tester: tester,
+        showHeader: true,
+        showFooter: true,
+        slide: const BigFactSlide(),
+      );
+      await slideTester.pumpSlide();
+
+      final socialHandleFinder = find.text('@flutter_deck');
+      final headerFinder = find.text('Slide');
+      final slideNumberFinder = find.text('1');
+      final bigFactFinder = find.text('100%');
+      final descriptionFinder = find.text('The test coverage value');
+
+      expect(socialHandleFinder, findsOneWidget);
+      expect(headerFinder, findsOneWidget);
+      expect(slideNumberFinder, findsOneWidget);
+      expect(bigFactFinder, findsOneWidget);
+      expect(descriptionFinder, findsOneWidget);
+    });
+
+    testWidgets('should apply theming', (WidgetTester tester) async {
+      final slideTester = SlideTester(
+        tester: tester,
+        showHeader: false,
+        showFooter: false,
+        slide: const BigFactSlide(),
+      );
+      await slideTester.pumpSlide();
+      final sliderFinder = find.byKey(const Key('BigFactslide'));
+      final slider = tester.widget(sliderFinder) as FlutterDeckSlide;
+      final textStyleFinder = slider.theme!.bigFactSlideTheme.titleTextStyle!;
```

Comment:
This is not a finder. You should rename it:
```suggestion
      final textStyle = slider.theme!.bigFactSlideTheme.titleTextStyle!;
```